.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
- [ ] Kindly remove this as it makes project unprofessional and unclean.

Reply:
@arslanbisharat  should I delete the whole folder or just that line?